notifications/src/main/java/dk/shape/games/notifications/actions/NotificationSettingsAction.kt

```diff
@@ -3,14 +3,26 @@ package dk.shape.games.notifications.actions
 import dk.shape.games.sportsbook.offerings.common.action.Action
 import kotlinx.android.parcel.Parcelize
 
-@Parcelize
-data class NotificationSettingsAction(
-    val betslipComponentUUID: String? = null,
-    val openedFromMyGames: Boolean = false,
-    val eventIds: List<String>? = null
-) : Action {
+sealed class NotificationSettingsAction : Action {
+
+    @Parcelize
+    data class NotificationsForAllEventIds(
+        val eventIds: List<String> = emptyList()
+    ) : NotificationSettingsAction()
+
+    @Parcelize
+    object FromMyGames : NotificationSettingsAction()
```

Comment:
```suggestion
    object FilterBetsEventIds : NotificationSettingsAction()
```
remove this one if not used anymore with the new bet history

Reply:
It has to be left for now, since the removal of unnecessary bits of code will come later. I'll deprecate it instead